docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -148,7 +191,7 @@ Review the current charts that are in use, so far:
 
 ```
 NAME               	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART                  	APP VERSION
-k8ssandra         	default  	1       	2020-11-16 20:17:23.107265 -0700 MST	deployed	k8ssandra-0.23.0        3.11.7  
+k8ssandra.         	default  	1       	2021-01-16 20:17:23.107265 -0700 MST	deployed	k8ssandra-0.9.0        	3.11.7  
```

Comment:
Should the period after _k8ssandra_ in the first column be there?

Reply:
good catch - removed